winrt/docsrc/CanvasFontFace.xml

```diff
@@ -280,6 +286,33 @@ Licensed under the MIT License. See LICENSE.txt in the project root for license
     </member>
+    <member name="T:Microsoft.Graphics.Canvas.Text.CanvasGlyphMetrics">
+      <summary>Represents the metrics of an individual glyph. The units depend on how the metrics are obtained.</summary>
+    </member>
+    <member name="F:Microsoft.Graphics.Canvas.Text.CanvasGlyphMetrics.LeftSideBearing">
+      <summary>Specifies the X offset from the glyph origin to the left edge of the black box. The glyph origin is the current horizontal writing position. A negative value means the black box extends to the left of the origin (often true for lowercase italic 'f').</summary>
+    </member>
+    <member name="F:Microsoft.Graphics.Canvas.Text.CanvasGlyphMetrics.AdvanceWidth">
+      <summary>Specifies the X offset from the origin of the current glyph to the origin of the next glyph when writing horizontally.</summary>
+    </member>
+    <member name="F:Microsoft.Graphics.Canvas.Text.CanvasGlyphMetrics.RightSideBearing">
+      <summary>Specifies the X offset from the right edge of the black box to the origin of the next glyph when writing horizontally. The value is negative when the right edge of the black box overhangs the layout box.</summary>
+    </member>
+    <member name="F:Microsoft.Graphics.Canvas.Text.CanvasGlyphMetrics.TopSideBearing">
+      <summary>Specifies the vertical offset from the vertical origin to the top of the black box. Thus, a positive value adds whitespace whereas a negative value means the glyph overhangs the top of the layout box.</summary>
+    </member>
+    <member name="F:Microsoft.Graphics.Canvas.Text.CanvasGlyphMetrics.AdvanceHeight">
+      <summary>Specifies the Y offset from the vertical origin of the current glyph to the vertical origin of the next glyph when writing vertically. Note that the term "origin" by itself denotes the horizontal origin. The vertical origin is different. Its Y coordinate is specified by verticalOriginY value, and its X coordinate is half the advanceWidth to the right of the horizontal origin.</summary>
+    </member>
+    <member name="F:Microsoft.Graphics.Canvas.Text.CanvasGlyphMetrics.BottomSideBearing">
+      <summary>Specifies the vertical distance from the bottom edge of the black box to the advance height. This is positive when the bottom edge of the black box is within the layout box, or negative when the bottom edge of black box overhangs the layout box.</summary>
+    </member>
+    <member name="F:Microsoft.Graphics.Canvas.Text.CanvasGlyphMetrics.VerticalOriginY">
+      <summary>Specifies the Y coordinate of a glyph's vertical origin, in the font's design coordinate system. The y coordinate of a glyph's vertical origin is the sum of the glyph's top side bearing and the top (that is, yMax) of the glyph's bounding box.</summary>
+    </member>
+    <member name="F:Microsoft.Graphics.Canvas.Text.CanvasGlyphMetrics.BlackBox">
+      <summary>Specifies the bounding box that encloses the glyph.</summary>
```

Comment:
Could this say, 'Specifies the bounds of the parts of the glyph are covered if the glyph is drawn.'.